CONTRIBUTING.rst

```diff
@@ -179,3 +179,19 @@ still be quite slow compared to normal unit tests. Nevertheless, if you are chan
 these tests, you should run them periodically.
 
 Do not commit the files ("large_cassettes/\*.yml") to the repository.
+
+Randomized tests
+~~~~~~~~~~~~~~~~
+
+Tests should be executable in any order. You can run this command
+a few times to verify if they are:
+
+    pytest --random-order
```

Comment:
Should we add this to the default options, or is it not stable enough yet?

Reply:
Not stable enough.